### DISCUSSION TITLE
Laravel Valet Driver for Prestashop

### DISCUSSION THREAD

**Discussion by toons43** [laravel/valet#1121] [Ideas]
Hello,

Would it be possible to create a driver for Prestashop 1.7.x ?
I tried multiple ways and managed to get the front office but coudn't make it work for the back office.

Thanks a lot in advance
Regards

**Comment by jadonbrownza** [comment]
Hi there, I have found https://github.com/markahesketh/prestashop-driver-laravel-valet to work for me.

**Comment by maengkom** [comment]
The latest prestashop always get `500 server error`.
